eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/EraIndepGenerators.hs

```diff
@@ -15,59 +16,25 @@
 
 module Test.Cardano.Ledger.Shelley.Serialisation.EraIndepGenerators
   ( genCoherentBlock,
```

Comment:
Maybe we should call this  Test.Cardano.Ledger.Shelley.Serialisation.Generators
since none of the type here are Era independent, now.  That is how the other Era  generators are consistently named:

module Test.Cardano.Ledger.ShelleyMA.Serialisation.Generators
module Test.Cardano.Ledger.Alonzo.Serialisation.Generators
module Test.Cardano.Ledger.Babbage.Serialisation.Generators